crates/cli/src/subcommands/generate/csharp.rs

```diff
@@ -706,11 +700,18 @@ pub fn autogen_csharp_globals(ctx: &GenCtx, items: &[GenItem], namespace: &str)
 
     writeln!(
         output,
-        "public class SpacetimeDBClient : SpacetimeDBClientBase<ReducerEvent>"
+        "public class DbConnection : DbConnectionBase<DbConnection, EventContext>"
     );
     indented_block(&mut output, |output| {
-        writeln!(output, "protected SpacetimeDBClient()");
+        writeln!(output, "public readonly RemoteTables RemoteTables = new();");
+        writeln!(output, "public readonly RemoteReducers RemoteReducers = new();");
```

Comment:
Shouldn't this entire file be excluded from this PR? IIUC you're making client-side changes in #1707 and that one seems a bit further along.

Reply:
Ah right it slipped in from the client sdk pr